src/components/BooksForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class BooksForm extends Component {
```

Comment:
- Move this component to the `src/containers` folder where it rightly makes sense
- Add a constant with book categories. Something like
```
const categories = ['code', 'comedy'], etc
```